### PR TITLE
Changed loader type to c4.2xlarge in 50gb

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
since c5 Are more prone to being shut down due to spot termination

## PR pre-checks (self review)

~~- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)~~
~~- [ ] I gave variables/functions meaningful self-explanatory names~~
~~- [ ] I didn't leave commented-out/debugging code~~
~~- [ ] I didn't copy-paste code~~
- [x] I added the relevant `backport` labels
~~- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)~~
~~- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
~~- [ ] I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
 # Seems the c5.xlarge type is small for this load - I receive OOM on 2 loaders
-instance_type_loader: 'c5.2xlarge'
```
Comment (amoskong) [inline test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml]: Why do you change this? can you add reason to commitlog?

Comment (roydahan) [reply test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml]: @ShlomiBalalis explain why in the commit message.

The reason is that we get spot termination for c5 very easily.

**Comment by fruch** [comment]
@roydahan I think we have more issue with spots right now, this alone won't fix them all ? right ?

**Comment by roydahan** [comment]
right, but this should help.

**Comment by bentsi** [comment]
@fruch @roydahan backport?

**Comment by roydahan** [comment]
Yes please :)

**Pull request merged**